practices/team_culture.md

```diff
@@ -0,0 +1,87 @@
+---
+slug: team-culture
+stages:
+  - building_team
+short_description: A team culture is how people work together towards a common goal with team members sharing their values, beliefs, attitudes and behaviours.
+tags:
+  - team culture
+  - team
+  - team members
+  - common goal
+  - values
+  - attitudes
+  - team work
+keywords:
+  - team culture
+  - team
+  - team members
+  - common goal
+  - values
+  - beliefs
+---
+
+# Team Culture
+
+**TL;DR**
+
+Team Culture is an essential part of any organization and it emphasizes on achieving a common goal by sharing values, beliefs, attitudes and behaviours among team members.
+
+## What Is Team Culture?
+
+Team Culture is not dependant on or created by an individual member's personality, but rather built on how the team collectively functions.
+
+The definition of team culture is fairly simple as it implies each member of a team should work towards achieving a common goal with collective efforts, discussions and feedback.
+
+## Why You Might Want Team Culture?
+
+An individual person working on the project is far less productive than a team working on the same project. It is very easy to understand that why a team will be more productive than an individual. An individual person has to do all the work during all stages of development.
+
+Team Culture is the change in traditional workplace culture. It is about achieving a goal through collective effort of all the team members rather than leaving all the burden on an individual person. Team work is very essential part of team culture in every aspect.
+
+_"Teamwork is the ability to work together toward a common vision. The ability to direct individual accomplishments toward organizational objectives. It is the fuel that allows common people to attain uncommon results." -Andrew Carnegie_
+
+![Team culture example](/files/team_culture.jpg)
+[Team culture example](https://pixabay.com/photos/work-office-team-company-internet-5382501/)
+
+## Problems Team Culture Helps to Solve
+
+- Performance
+- Efficiency
+- Aggressive Competition
+- Demotivated Individuals
+- Project Deadline
+- [Meaningless work](/problems/meaningless-work)
+- Lack of ideas
```

Comment:
I like this, but I would suggest adjusting this. From where this lack of ideas comes from? Well, from lack of communication and the lack of values. We (simplified) act on our values. What do you think about it?

Reply:
Yeah , I got it.